templates/ts-apollo-fullstack/server/src/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { loadConfigSync } from 'graphql-config';
+import Knex from 'knex';
+
+export function getProjectConfig() {
+  const config = loadConfigSync({
+    rootDir: process.cwd(),
```

Comment:
this is also where also loaders are supplied so we can have more control how we load config and schema